internal/argsbuilder/argsbuilder_interface.go

```diff
@@ -7,6 +7,7 @@ type ArgsBuilder interface {
 	SetContainerName(name string) ArgsBuilder
 	SetNetworkMode(networkMode string) ArgsBuilder
 	SetPrivileged(privileged bool) ArgsBuilder
+	SetReadOnlyRoot(readOnlyRootfs *bool) ArgsBuilder
```

Comment:
Are there any other options for which we should be implementing tri-state logic?  (Like all of them?  😇)

(Also, the handling of `-e` looks...incomplete -- it seems like it silently ignores cases with no `=` as well as cases where the _value_ of the environment variable contains a `=`....)